containers/ecr-viewer/guide.md

```diff
@@ -0,0 +1,145 @@
+  user -->|<code>/</code><br>eCR Library<br>Authenticated By identity provider| container
+  ingestion -->|<code>/api/save-fhir-data</code>| container
+```
+
+### Dual Architecture Diagram
+
+```mermaid
+flowchart LR
+  user
+  nbs["NBS - View eICR"]
+  ingestion["Ingestion Service<br>(e.g. Rhapsody)"]
+  subgraph service[Cloud]
+    direction TB
+    subgraph container["fab:fa-docker container"]
+      viewer["fab:fa-node fab:fa-react <code>ecr-viewer<br>HTTP:3000/</code>"]
+    end
+    fileStorage["fab:fa-file File Storage"]
+    metadata["fab:fa-database Database"]
+    identityProvider["Identity Provider<br>(e.g. Keycloak/Azure Ad)"]
+	container <--> |eCR FHIR Data| fileStorage
+	container <--> |eCR Metadata| metadata
+    container -->|Validates User| identityProvider
+  end
+  user --> nbs
+  nbs -->|<code>/view-data</code><br>eCR Viewer<br>Authenticated by NBS| container
+  user -->|<code>/view-data</code><br>eCR Viewer<br>Authenticated by identity provider| container
+  user -->|<code>/</code><br>eCR Library<br>Authenticated By identity provider| container
+  ingestion -->|<code>/api/save-fhir-data</code>| container
+```
+
+## Environment Variable Setup
+
+The full list of environment variables can be found in {@link NodeJS.ProcessEnv}
+
+### Base Required
+
+Base required variables are ones required for all deployments regardless of mode or cloud environment. If variables are not set, this may cause issues starting the app. The variables can be found in {@link EnvironmentVariables.BaseRequired}.
+
+### eCR Fhir Storage
+
+A storage container for the eCRs must be created for all deployments. Depending on the storage container used additional variables may be required. The variables can be found in {@link EnvironmentVariables.EcrStorage}.
+
+### Authentication Setup
+
+Authentication is required when running any mode modes of the application.
+
+#### Integrated Authentication
+
+Integrated eCR Viewer will rely on NBS to authenticate the user. The variables can be found in {@link EnvironmentVariables.Authentication}.
+
+#### Non-Integrated Authentication
+
+Non-Integrated/Dual rely on an external authentication provider, like azure ad, entra, or keycloak. The variables can be found in {@link EnvironmentVariables.Authentication}.
+
+### eCR Metadata Storage
+
+In order to store metadata for the eCR Library, an eCR metadata database must be created. The variables can be found in {@link EnvironmentVariables.EcrMetadataStorage}
+
+### Removed Environment Variables
+
+These are variables that have been retired and no longer have a use in the app. These can be safely removed when installing the current version.
+
+| Name | Description | Version Removed | Date Removed |
+| ---- | ----------- | --------------- | ------------ |
+|      |             |                 |              |
+
+## Inserting data
+
+### From Rhapsody
+
+Data can be added to the eCR Viewer as a step in Rhapsody.
+
+```js
+Placeholder script
```

Comment:
Do we want to just link to another file stored in Github? Either a Rhapsody file or the native JavaScript?

Reply:
We link to everything else here, so it makes sense to link I think? We could also just include a code snippet by wrapping the code in `#region rhapsody `and `#endregion rhapsody`